test/cpp/interop/metrics_client.cc

```diff
@@ -88,7 +88,7 @@ bool PrintMetrics(std::unique_ptr<MetricsService::Stub> stub, bool total_only,
 int main(int argc, char** argv) {
   grpc::testing::InitTest(&argc, &argv, true);
 
-  // The output of metrics client is in some cases programatically parsed (for
+  // The output of metrics client is in some cases programmaticaly parsed (for
```

Comment:
programmatically?

Reply:
Thank you very much, this has been updated as your recommendation :)